docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -205,9 +242,19 @@ It should be tested primarily in isolation as other components shouldn’t affec
 
 ### Graduation Criteria [optional]
 
-alpha - The feature is initially constructed and toggleable with an environment variable
-beta - The feature has e2e tests implemented and is more integrated and seamless with the project.
-At this point we should also move the
+Alpha:
+
+The feature is initially constructed and toggleable with an environment variable
+
+Beta:
+
+The feature has e2e tests implemented and is more integrated and seamless with the project.
+
+Stable: 
+
+The feature has been used for a while and is widely acceptable as well as reliable and will now be enabled by default.
+
+For AzureMachineTemplate we will move the feature enabled specification from metadata.annotations to a new optional field under spec.template.spec called nodeCachingMode which will be enabled if present and a sub field under that called cacheInterval which will be set to 1 day by default, and it will be a required subfield of nodeCachingMode if that field is there. For AzureMachinePool we will do the same thing under spec.template in the AzureMachinePool files.
```

Comment:
It may be worth noting that CAPZ should still interpret the annotation for some TBD number of releases to allow users time to transition to the `spec` field.

Reply:
Ended up just moving it altogether and putting it behind a feature flag